release/smoke-tests/otel-span-exporter/requirements.txt

```diff
@@ -0,0 +1,22 @@
+backoff==1.10.0
```

Comment:
Is there anyway to reduce this list? This script took over 6 minutes to start for me and it appears that much of the time was spent in installing Python requirements.

Reply:
Most of the time is spent compiling grpcio dependencies for alpine. Using a larger base image to reduce build time is an option.

I was able to remove some dependencies but the list will remain long to version lock all dependencies and children.

Reply:
Thanks! Maybe we can optimize this later.